src/dart/parse.rs

```diff
@@ -114,6 +119,8 @@ impl<'a> Parser<'a> {
         }
     }
 
+    /// Consumes the current token and the whitespaces until the following
+    /// non-whitespace token.
```

Comment:
"and any number of whitespace tokens" - also, "consumes" is not ideal.
Maybe "Advances the current token, skipping past any number of whitespace tokens."?